### PR TITLE
add rds config for hmpps-interventions-dev

### PR DESCRIPTION
**Pull request by tomsmyers**

### CODE REVIEW

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-interventions-dev/resources/rds.tf [hunk 1]
```diff
@@ -0,0 +1,36 @@
+module "hmpps_interventions_rds" {
+  source                 = "github.com/ministryofjustice/cloud-platform-terraform-rds-instance?ref=5.10"
+  cluster_name           = var.cluster_name
+  cluster_state_bucket   = var.cluster_state_bucket
+  team_name              = var.team_name
+  business-unit          = var.business_unit
+  application            = var.application
+  is-production          = var.is_production
+  namespace              = var.namespace
+  environment-name       = var.environment_name
```
Comment (sldblog) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-interventions-dev/resources/rds.tf:10]: Inline from https://concourse.cloud-platform.service.justice.gov.uk/builds/6916
> Error: Reference to undeclared input variable
>
>  on rds.tf line 10, in module "hmpps_interventions_rds":
>  10:   environment-name       = var.environment_name

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-interventions-dev/resources/variables.tf [hunk 2]
```diff
@@ -46,3 +46,7 @@ variable "slack_channel" {
 variable "number_cache_clusters" {
   default = "2"
 }
+
+variable "rds_family" {
+  default = "postgres11"
```
[the remark this answers is not in the document]
Comment (tomsmyers) [reply namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-interventions-dev/resources/variables.tf:51]: not according to this 

https://github.com/ministryofjustice/cloud-platform-terraform-rds-instance/blob/b87547d9470b1d299cfe803b7777954e15dccc89/template/rds.tmpl#L20

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-interventions-dev/resources/rds.tf [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  rds_family             = var.rds_family
+
+  providers = {
+    aws = aws.london
+  }
+}
+
+resource "kubernetes_secret" "hmpps_interventions_rds" {
+  metadata {
+    name      = "hmpps-interventions-rds-instance-output"
```
[the remark this answers is not in the document]
Comment (tomsmyers) [reply namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-interventions-dev/resources/rds.tf:21]: sure - this was a common pattern from other modules, but happy to change

**Review by sldblog — APPROVED** [review]

**Comment by sldblog** [comment]
reopened to kick off stuck CI

**Review by jasonBirchall — APPROVED** [review]

**Pull request merged**